client/my-sites/plugins/controller.js

```diff
@@ -204,3 +205,10 @@ export function scrollTopIfNoHash( context, next ) {
 	}
 	next();
 }
+
+export function renderDomainsPage( context, next ) {
+	context.primary = (
+		<AsyncLoad require="calypso/my-sites/plugins/marketplace/marketplace-domain-upsell" />
```

Comment:
Nit: have we considered the default placeholder that appears here for just a little while this is loading? We often remove it by providing `placeholder={ null }`

Reply:
I didn't know this, but I would opt to leave the default placeholder for now and circle back if stakeholders opt to remove it. In any case it feels better without.